src/probeinterface/probe.py

```diff
@@ -143,7 +144,7 @@ def model_name(self):
 
     @model_name.setter
     def model_name(self, value):
-        if value is not None:
+        if value is not None:  # Alessio, why propagate to annotations and not just set the attribute?
```

Comment:
```suggestion
        if value is not None:
```

Reply:
Will remove! I just wanted to call your attention in the diff : )

Reply:
Done.